docs/configuration.md

```diff
@@ -227,9 +227,9 @@ Defines some pre-configured world settings for the server.
 - COMPLETE_DARKNESS
 - DST_CAVE
 
-**WORLD_OVERRIDES**  
-Sets the overrides-configuration for world generation. Basically it's just the content for the
-`worldgenoverride.lua` file. As this value can be pretty large it's recommended to put the
+**LEVELDATA_OVERRIDES**  
```

Comment:
You changued the variable name in the docs but the code doesn't use `LEVELDATA_OVERRIDES` it continues using `WORLD_OVERRIDES`.

Probably you must find all the ocurrences of `WORLD_OVERRIDES` and renamed. This is a Breaking Change... but Klei introduced it so I think that it's ok to do it.